ambry-metrics/src/main/java/com.github.ambry.metrics/MetricsRegistryMap.java

```diff
@@ -1,115 +0,0 @@
-/**
- * Copyright 2016 LinkedIn Corp. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
```

Comment:
This was a part of the `StoreManager` PR whose commit was removed - Casey has just put up a PR reapplying that commit - should go away once that is merged